src/tokenized-buffer.coffee

```diff
@@ -349,13 +349,23 @@ class TokenizedBuffer extends Model
 
   indentLevelForLine: (line) ->
     if match = line.match(/^[\t ]+/)
-      leadingWhitespace = match[0]
-      tabCount = leadingWhitespace.match(/\t/g)?.length ? 0
-      spaceCount = leadingWhitespace.match(/[ ]/g)?.length ? 0
-      tabCount + (spaceCount / @getTabLength())
+      {indentLevel} = @parseIndentation(match[0])
+      indentLevel
     else
       0
 
+  parseIndentation: (leadingWhitespace) ->
+    tabLength = @getTabLength()
+    columnCount = 0
+    for c in leadingWhitespace
+      if c == '\t'
```

Comment:
Recommendation: use `is` instead of `==`. See [this section](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#coffeescript-styleguide) of the contributing guide.